### PR TITLE
Add ECS integration tests.

### PR DESCRIPTION
**Pull request by swaggy**
@mikhail

### CODE REVIEW

File: kingpin/actors/aws/test/integration_ecs.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    their definition order in this file. Nose follows this order according
+    to its documentation:
+
+        http://nose.readthedocs.org/en/latest/writing_tests.html
+    """
+
+    integration = True
+
+    region = 'us-west-2'
+    cluster = 'staging-us1'
```
Comment (diranged) [inline kingpin/actors/aws/test/integration_ecs.py]: So don't hard code our own cluster in here. Make this a generic cluster name, but make it overridable with an environment variable. Same with the region.

File: kingpin/actors/aws/test/integration_ecs.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                'Service %s with count %s' % (name, count),
+                {'region': self.region,
+                 'cluster': self.cluster,
+                 'task_definition': definition,
+                 'service_name': '%s_%s' % (name, suffix),
+                 'state': state,
+                 'count': count})
+        return actor
+
+    def _successful_task(self):
```
Comment (diranged) [inline kingpin/actors/aws/test/integration_ecs.py]: I'm not sure of the value of this layout here... it just means I have to scroll up and down to read the code.

**Comment by swaggy** [comment]
I've addressed your comments.

**Comment by diranged** [comment]
👍 

do they work in jenkins?

**Comment by swaggy** [comment]
They worked locally, haven't tested in Jenkins.

**Comment by swaggy** [comment]
https://build.corp.nextdoor.com/job/systems-kingpin-continuous-build/10758/console

**Comment by swaggy** [comment]
They passed, I'm merging.

**Comment by diranged** [comment]
@swaggy  gonna merge this? :)

**Comment by swaggy** [comment]
Heh. Yeah.

**Pull request merged**